docs/source/sci_api_tutorial/armer.rst

```diff
@@ -2,21 +2,297 @@
+
+        `node_cores`: How many cores needed for each node. If GPU is used, usually only one core per node will be requested.
+
+        `job_name`: This sets the name of the job to "job_name". You can change this to a more descriptive name for your job.
+
+        `partition`: This specifies that the job should be submitted to a specific partition, which is likely a partition dedicated to GPU resources.
+        
+        `mem_per_core`: This requests a number of gigabytes of memory per core.
+        
+        `walltime`: This sets the maximum walltime (execution time) for the job. '24:00:00' means 24 hours.
+        
+        `account``: This specifies the account to be charged for the job's resource usage. 
+
+
+The correct input for these parameters should refer to the guidelines for each institution's HPC submission syntax. If you need to support your local HPC cluster, please refer to the `Quick Start: 2. Support Your Local Cluster` page from the menu bar on the left.
+
+
+How ARMer is used in EnzyHTP APIs
+------------------------------------------------
+
+.. dropdown:: :fa:`eye,mr-1` Developer Integration with Science APIs`
+
+    Developers using EnzyHTP can directly leverage ARMer's capabilities through the platform's APIs. 
+    
+    In EnzyHTP, ARMer is integrated into the workflow, which includes sub-tasks like mutant generation, molecular dynamics simulations, quantum mechanical calculations, and data analysis. 
+    Each subtask was a seperate science API that has different computing needs, which ARMer manages effectively.
+
+    Developers can utilize ARMer to tailor the computational resources specifically for the task at hand, whether it involves intensive CPU usage for molecular dynamics simulations or GPU resources for more complex quantum mechanical calculations.
+
+.. dropdown:: :fa:`eye,mr-1` User Interaction via Configured API`
+
+    For users, the interaction with ARMer is streamlined through configurations exposed by API developers:
+
+    Cluster Job Configuration Dictionary: This dictionary (cluster_job_config) shown above is provided by the API developers and exposes various configurable options that users can set according to their specific requirements. It includes parameters such as cluster type, environmental settings, and resource keywords.
+    
+    Simplified Job Submission: Users don’t need to manage complex cluster configurations directly. Instead, they provide necessary parameters through a high-level interface, simplifying the computational aspects of enzyme modeling.
+
+
+Input
+------------------------------------------------
+    In EnzyHTP, the cluster is an object that represents a specific HPC cluster configuration. This object is usually an instance of a class that implements the ClusterInterface or a similar interface that ARMer can interact with.
+    The input for ARMer should be all kinds of "clusters", for example, clusters for MD sampling, clusters for QM caluclations, etc.
```

Comment:
The cluster is not the input but one value under the "cluster" key of the cluster_job_config. Please downsize this and merge it with the corresponding section earlier about the key